lib/config/rules/style.js

```diff
@@ -0,0 +1,58 @@
+// Style:
+// These rules enforce consistent style across your codebase
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Enforces vertical alignment.
+  'align': true,
+  // Requires using either ‘T[]’ or ‘Array' for arrays.
+  'array-type': false,
+  // Requires parentheses around the parameters of arrow function definitions.
+  'arrow-parens': true,
+  // An interface or literal type with just a call signature can be written as a function type.
+  'callable-types': false,
+  // Enforces PascalCased class and interface names.
+  'class-name': true,
+  // Enforces formatting rules for single-line comments.
+  'comment-format': [true, 'check-space'],
+  // Enforces documentation for important items be filled out.
+  'completed-docs': false,
+  // Enforces a certain header comment for all files, matched by a regular expression.
+  'file-header': false,
+  // Ensures proper spacing between import statement keywords
+  'import-spacing': true,
+  // Requires interface names to begin with a capital ‘I’
+  'interface-name': false,
+  // Prefer an interface declaration over a type literal (`type T = { ... }`)
+  'interface-over-type-literal': true,
+  // Enforces basic format rules for JSDoc comments.
+  'jsdoc-format': true,
+  // Requires parentheses when invoking a constructor via the `new` keyword.
+  'new-parens': true,
+  // Requires the use of `as Type` for type assertions instead of `<Type>`.
+  'no-angle-bracket-type-assertion': true,
+  // Disallows one or more blank lines in a row.
+  'no-consecutive-blank-lines': [true, 2],
+  // Disallows parameter properties in class constructors.
+  'no-parameter-properties': false,
+  // Enforces consistent object literal property quote style.
+  'object-literal-key-quotes': [true, 'as-needed'],
+  // Enforces use of ES6 object literal shorthand when possible.
+  'object-literal-shorthand': true,
+  // Requires the specified tokens to be on the same line as the expression preceding them.
+  'one-line': [true, 'check-catch', 'check-finally', 'check-else'],
```

Comment:
Good catch. We should keep `check-open-brace` on. This checks to ensures that we have
``` 
if (true) {
  bar();
}
```

instead of 
``` 
if (true)
{
  bar();
}
```

We should keep `check-whitespace` on too. It ensures that we have spaces proceeding specified tokens.
``` 
if (true) {
  bar();
}
```